packages/rum-core/package.json

```diff
@@ -17,11 +17,13 @@
   "scripts": {
     "prepublishOnly": "npm run build:main",
     "build:main": "npx babel src -d dist/lib",
+    "karma:dev": "karma start --auto-watch --restartOnFileChange",
+    "karma:bench": "NODE_ENV=production karma start karma.bench.conf.js",
```

Comment:
should be `bench` because of how benchmark script is run https://github.com/elastic/apm-agent-rum-js/blob/c38f175a3d9b8352c2d153318eb2ccf5c67e91c7/scripts/benchmarks.js#L90